client/styles/SettingsPage.module.css

```diff
@@ -0,0 +1,54 @@
+.settings-form-container {
+  padding: 20px 16px;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.settings-page-title {
+  margin-bottom: 16px;
+  align-self: center;
+  font-size: 40px;
+}
+
+.settings-field-group {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  align-items: center;
+}
+.settings-field-label {
+  text-align: left;
+  align-self: flex-start;
+  font-size: 20px;
+}
+
+.settings-text-field :global(.MuiOutlinedInput-root) {
```

Comment:
Personally, I find the border styling stands out quite a bit. I noticed that in other apps I've used with settings features, the text input border is normally less pronounced when not focused. 

Here are a couple of examples:
![image](https://github.com/user-attachments/assets/6c0a6cf5-395f-4fb6-8aa9-b13be5fd8bce)
![image](https://github.com/user-attachments/assets/f023ced2-e1ee-470e-8e22-12c0248adb80)